Presentation.Web/Controllers/API/V2/Internal/OrganizationUnits/OrganizationUnitLocalRoleOptionTypesInternalV2Controller.cs

```diff
@@ -0,0 +1,92 @@
+﻿using Core.ApplicationServices.LocalOptions;
+using Core.DomainModel.LocalOptions;
+using Core.DomainModel.Organization;
+using Presentation.Web.Controllers.API.V2.Common.Mapping;
+using Presentation.Web.Infrastructure.Attributes;
+using Swashbuckle.Swagger.Annotations;
+using System.Collections.Generic;
+using System.Net;
+using System.Web.Http;
+using System;
+using Presentation.Web.Controllers.API.V2.Internal.LocalOptionTypes;
+using Presentation.Web.Models.API.V2.Internal.Response.LocalOptions;
+using Presentation.Web.Models.API.V2.Internal.Request.Options;
+
+namespace Presentation.Web.Controllers.API.V2.Internal.OrganizationUnits
+{
+    [RoutePrefix("api/v2/internal/organization-units/{organizationUuid}/local-option-types")]
+    public class OrganizationUnitLocalRoleOptionTypesInternalV2Controller: BaseLocalRoleOptionTypesInternalV2Controller<LocalOrganizationUnitRole, OrganizationUnitRight, OrganizationUnitRole>
+    {
+        private readonly IGenericLocalOptionsService<LocalOrganizationUnitRole, OrganizationUnitRight, OrganizationUnitRole> _localOrganizationUnitRoleOptionTypeService;
```

Comment:
same thing here with the service.

Also, do mappers need fields in this one? I think it's only used in the base class